static/styles/zulip.scss

```diff
@@ -541,13 +541,12 @@ strong {
 li.actual-dropdown-menu i {
     /* In gear menu, make icons the same width so labels line up. */
     display: inline-block;
-    width: 14px;
+    width: auto;
     text-align: center;
-    margin-right: 3px;
 }
```

Comment:
These hunks mess up the alignment of the menu described in the comment for this block; I removed them.